.github/workflows/continuous-deployment.yml

```diff
@@ -35,7 +35,7 @@ jobs:
 
             - name: Get pnpm store directory
               id: pnpm-cache
-              run: echo "::set-output name=pnpm_cache_dir::$(pnpm store path)"
+              run: echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
```

Comment:
Doesn't the `$GITHUB_OUTPUT` need to be wrapped in `"`? Looks like this one and `$GITHUB_ENV` use them.  
See doc here:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
Works the same, the data is still passed https://github.com/Frontify/fondue/actions/runs/6025904700/job/16347675219#step:6:3